src/backlight/portfolio/portfolio.py

```diff
@@ -0,0 +1,127 @@
+import pandas as pd
+import numpy as np
+from typing import List
+
+from backlight.positions.positions import Positions
+from backlight.datasource.marketdata import MarketData
+from backlight.metrics.position_metrics import calc_pl
+from backlight.positions import calc_positions
+from backlight.trades.trades import Trades
+from joblib import Parallel, delayed
+
+
+class Portfolio:
+    """
+    An abstract definition of a Portfolio is defined as a list of Positions
+    Each element of the list represent the positions of an asset that contribute to the portfolio
+    """
+
+    def __init__(self, positions: List[Positions]):
+        self.positions = positions
+
+    def value(self) -> pd.DataFrame:
+        """ DataFrame of the portfolio valuation of each asset"""
+        pl = pd.DataFrame()
+        for asset_positions in self.positions:
+            # Compute PL of positions of each asset
+            pl[asset_positions.symbol] = calc_pl(asset_positions)
+        return pl
```

Comment:
How the result will become when each position's indexes are not same?
Please create a ticket to make it compatible with different frequency of positions.